javascripts/discourse/api-initializers/modifications_for_google_analytics.js

```diff
@@ -1,8 +1,8 @@
 import { apiInitializer } from "discourse/lib/api";
-import { getOwner } from "discourse-common/lib/get-owner";
 
 export default apiInitializer("0.11.1", api => {
-  const currentUser = getOwner(this).lookup("service:current-user");
+  const router = api.container.lookup("service:router");
+  const currentUser = router.currentUser;
```

Comment:
The deprecation refers to the import, the `current-user` service itself is valid:

```suggestion
  const currentUser = api.container.lookup("service:current-user");
```